src/Database/Log/LoggedQuery.php

```diff
@@ -73,7 +82,12 @@ protected function interpolate(): string
             if ($p === null) {
                 return 'NULL';
             }
+
             if (is_bool($p)) {
+                if ($this->driver instanceof Postgres || $this->driver instanceof Sqlite) {
```

Comment:
Wouldn't it be better to check for `Mysql` and return 0/1 and default to `TRUE`/`FALSE`?

Reply:
I think MySQL will also accept TRUE/FALSE.

Reply:
SqlServer uses `bit` which accepts only 1 or the string 'TRUE' and not the literal TRUE. It was an even split.

Reply:
Just tested and you are right that MySQL accepts `TRUE` literal.